README.md

```diff
@@ -5,11 +5,12 @@ When considering each of these technologies, provide one or two sentences on wha
  * How does device introduce itself to the infrastructure or peer?
  * What, if any, proof of possession mechanism is there?
  * What form of credential is returned?
- * Is online access required for onboarding?
+ * Is full Internet access required for onboarding?
```

Comment:
It's also important to ask whether local IP access is required for onboarding.  (Since if the answer is yes, then generally such an onboarding solution is not sufficient since you have to have another solution to onboard it at L2.)